Disable "main_context_futures" tests on armel, they cause the testsuite
to fail erratically with "signal: 6, SIGABRT: process abort signal",
there is no obvious link between cause and effect, the problem was
narrowed down by painful trial and error.

The arm "target feature" flags are not available in standard rust,
so we identify armel by the fact has a target_arch of arm and does
not have 64-bit atomics.

Index: rust-glib-0.17.10/src/main_context_futures.rs
===================================================================
--- rust-glib-0.17.10.orig/src/main_context_futures.rs
+++ rust-glib-0.17.10/src/main_context_futures.rs
@@ -687,6 +687,7 @@ impl LocalSpawn for MainContext {
 }
 
 #[cfg(test)]
+#[cfg(any(not(target_arch = "arm"),target_has_atomic = "64"))]
 mod tests {
     use std::{sync::mpsc, thread};
 
